guides/user/pipeline/expressions/index.md

```diff
@@ -37,28 +40,27 @@ ${expression here}
 If an expression can't be evaluated, Spinnaker simply returns the expression.
 The above example would return "expression here."
 
-You also can interpolate expressions and strings. For example if
-`expressionA` evaluates to _Hello_ and `expressionB` evaluates to _world_,
+You also can interpolate expressions and strings. For example, if
 
 ```
  ${expressionA}-randomString-${expressionB}
 ```
 
 evaluates to _Hello-randomString-world_.
 
-Note that expressions can't be nested: `${ expression1 ${expression2} }` won't
-be evaluated.
+  > __Note:__ Expressions can't be nested: `${ expression1 ${expression2} }` won't
+  > be evaluated.
 
 ## How do I use a pipeline expression?
 
 ### Where can I use pipeline expressions?
 
-You can use pipeline expressions any place in the UI where you can enter
+You can use pipeline expressions in any place in the UI where you can enter
```

Comment:
For this one, I think, the original's better too. But it might sound more right if you changed "any place" to "anywhere."